contracts/bonding/BondingManager.sol

```diff
@@ -287,7 +282,8 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         if (delegatorStatus(msg.sender) == DelegatorStatus.Unbonded) {
             // Withdraw bonded amount which is now unbonded
             amount = amount.add(delegators[msg.sender].bondedAmount);
-            delete delegators[msg.sender];
+            delegators[msg.sender].bondedAmount = 0;
+            delegators[msg.sender].withdrawRound = 0;
```

Comment:
Should we also set the `delegatedAmount` to 0?

Reply:
Didn't set `delegatedAmount` to 0 here for consistency - if delegator A is delegated to delegator B and delegator B withdraws, delegator A is still delegated to delegator B since we don't have a way to automatically remove delegator A's delegate. Since delegator A remains delegated to delegator B we can keep `delegatedAmount` for delegator B and only change it if delegator A bonds to another delegator or unbonds